src/Accelerators/NNPA/Conversion/ONNXToZHigh/PerfModel.inc

```diff
@@ -0,0 +1,216 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+//===----------------- Auto-Generated, do not change  ---------------------===//
```

Comment:
Could you give additional comments on how to generate this? e.g. scripts, commands